packages/runner/src/bin/start-runner.ts

```diff
@@ -12,22 +13,22 @@ const instanceId = process.env.INSTANCE_ID;
 
 if (!instancesServerPort || instancesServerPort !== parseInt(instancesServerPort, 10).toString()) {
     console.error("Incorrect run argument: instancesServerPort");
-    process.exit(1);
+    process.exit(RunnerExitCode.INVALID_ENV_VARS);
```

Comment:
@pietrzakacper providing exit code this way is deprecated, it should be done via `process.exitCode = `

Reply:
I dont think it's deprecated. Setting a property is just a way for setting exitCode when we don't want to exit forcefully. https://nodejs.org/api/process.html#processexitcode

Reply:
In this case, we want to

Reply:
Oki. I was mistaken then. :)